### PR TITLE
feat: forward partnerContentfulPageId setting

### PR DESCRIPTION
**Pull request by michelleinez**

### CODE REVIEW

**Review by emuvente — APPROVED** [review]

File: server/auth-router.js [hunk 1]
```diff
@@ -119,6 +119,11 @@ module.exports = function authRouter(config = {}) {
 			options.passwordless = true;
 		}
 
+		// Specify partnerContentfulPageId
+		if (req.query.partnerContentfulPageId) {
```
Comment (mcstover) [inline server/auth-router.js:123]: Let's shorten this to `partnerContentId` please.

Comment (michelleinez) [reply server/auth-router.js]: done

**Review by mcstover — APPROVED** [review]
